rise/static/main.js

```diff
@@ -619,6 +620,13 @@ function setup() {
     return true;
   };
   $(document).keydown(document_keydown);
+
+  // autolaunch if specified in metadata
+  var config = configSlides()
+  var autolaunch = config.get_sync('autolaunch');
```

Comment:
I would suggest using a pattern like this one: https://github.com/damianavila/RISE/blob/master/rise/static/main.js#L191-L193

You can see there I used a promise... I found that way to be more reliable at the time to load config file values outside the Revealer function.